site/content/integrate/apps/_index.md

```diff
@@ -41,6 +41,11 @@ We have quick start guides for [TypeScript]({{< ref "quickstart/quick-start-ts"
 
 ## FAQ
 
+### What version of Mattermost is needed?
+    
+For v1.0 of the apps framework, the minimum Mattermost Server version is v6.6.
+For v1.2.0 of the apps framework, Mattermost Server v7.2 or later is needed.
```

Comment:
For versions 1.2.1 It was bumped to 7.7.0 in this PR [pr-mattermost-plugin-apps-449](https://github.com/mattermost/mattermost-plugin-apps/pull/449)